modules/spo-configuring-webhooks.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-configuring-webhooks_{context}"]
+= Configuring webhooks
+
+Both profile binding and profile recording make use of webhooks. Their configuration (an instance of `MutatingWebhookConfiguration` CR) is managed by SPO itself and is not part of the deployed YAML manifests. While the defaults should be acceptable for the majority of users and the webhooks do nothing unless an instance of either `ProfileBinding` or `ProfileRecording` exists in a namespace, it might still be useful to configure the webhooks.
+
+To change the configuration of the webhook, the `spod` CR exposes a `webhookOptions` object that allows the `failurePolicy` and the `namespaceSelector` variables to be set. This allows you to set the webhooks to "soft-fail" or restrict them to a subset of a namespaces so that even if the webhooks failed, other namespaces or resources are not affected.
+
+.Procedure
+
+. Set the `binding.spo.io` webhook configuration to ignore errors and restrict it to a subset of namespaces labeled with `spo.x-k8s.io/enable-binding=true` by creating the following patch file:
++
+[source,yaml]
+----
+spec:
+  webhookOptions:
+    - name: binding.spo.io
+      failurePolicy: Ignore
+      namespaceSelector:
+        matchExpressions:
+          - key: spo.x-k8s.io/enable-binding
+            operator: In
+            values:
+              - 'true'
+----
+
+. Patch the `spod/spod` instance by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-security-profiles patch spod \
+spod -p $(cat /tmp/spod-wh.patch) --type=merge
+----
+
+. To view the resulting `MutatingWebhookConfiguration`, run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-security-profiles get MutatingWebhookConfiguration \
```

Comment:
Let's drop the namespace flag here (this is actually a bug in the upstream docs and I just opened https://github.com/kubernetes-sigs/security-profiles-operator/pull/1314 to fix that)